src/style.css

```diff
@@ -1,3 +1,61 @@
+:root {
+  --link-color: #00ACDC;
+  --header-height: 60px
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  height: var(--header-height);
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav_list{
+  padding: 0;
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav_line {
+  display: flex;
+  position: relative;
+  align-items: center;
+  height: var(--header-height);
+}
```

Comment:
The class '.nav_line' does not follow BEM naming conventions. It should be named as an element of the '.nav' block, such as '.nav__line', assuming '.nav' is the block in this context.